model/models.js

```diff
@@ -50,23 +50,34 @@ exports.insertComment = (article_id, username, body) => {
   }
 
   return db
-    .query("SELECT * FROM articles WHERE article_id = $1", [article_id])
+    .query("SELECT * FROM users WHERE username = $1", [username])
```

Comment:
It's not necessary to query the users table or the articles table as they are both _foreign keys_ in the comments table, so you can make one query all to the comments table and if either are not found PSQL will let you know
_do note that might also require an amendment to your error handling middleware to handle the errors_